internal/repository/evmreader.go

```diff
@@ -239,3 +245,130 @@ func (pg *Database) GetLastProcessedBlock(
 
 	return block, nil
 }
+
+func (pg *Database) GetPreviousSubmittedClaimsEpochs(
```

Comment:
Please add some documentation to the function and rename it to
```suggestion
func (pg *Database) GetClaimSubmittedEpochs(
```

and fix the name used in the error messages below. Example:
https://github.com/cartesi/rollups-node/pull/587/files?diff=unified&w=0#diff-e4c6cf8eaabd1c2b1101fd6881c35efcd8ab6f4a07bbe52148cd947d80335c1fR279